docs/serving/samples/hello-world/helloworld-csharp/helloworld-csharp.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <RootNamespace>helloworld_csharp</RootNamespace>

```

Comment:
In this case, because the root namespace in the source code is already `helloworld_csharp`, this configuration is redundant and shouldn't be added.

See https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.localization.rootnamespaceattribute.rootnamespace?view=aspnetcore-2.2.